multiplexer/abci/remote_v1.go

```diff
@@ -173,8 +173,7 @@ func (a *RemoteABCIClientV1) FinalizeBlock(req *abciv2.RequestFinalizeBlock) (*a
 	}
 	events = append(events, abciEventV1ToV2(endBlockResp.Events...)...)
 
-	// convert tx results
-	txResults := make([]*abciv2.ExecTxResult, 0, len(commitBlockResps))
```

Comment:
This fixes an issue I found _after_ the script started working. I introduced it recently. The length of the slice should not be set to `0`, it should be set to `len(commitBlockResps)`

Reply:
nice catch, why was this working before? I would expect a panic on indexing at `0`.

Reply:
oh that was the linter change that you mentioned in the description, yeah that's a pretty weird one the linter made that change 🤔

Reply:
Linter didn't make the change, it just flagged that I needed to use an allocated list and then I made the bug when trying to address the linter.